components/movementsensor/adxl345/adxl345.go

```diff
@@ -189,10 +235,90 @@ func NewAdxl345(
 			}
 		}
 	})
-
```

Comment:
Again, I'd put back a blank line here, because the code above this and the code below this do separate tasks. I'd also put a blank line between lines 244 and 245 for similar reasons.